tests/e2e/upgrade/webapp.wdio-spec.js

```diff
@@ -6,35 +6,35 @@ const utils = require('@utils');
 const loginPage = require('@page-objects/default/login/login.wdio.page');
 const constants = require('@constants');
 
-const docs = [
-  {
+const placeFactory = require('@factories/cht/contacts/place');
+const personFactory = require('@factories/cht/contacts/person');
+const genericReportFactory = require('@factories/cht/reports/generic-report');
+
+describe('Webapp after upgrade', () => {
+  const district = placeFactory.place().build({
     _id: 'parent',
     type: 'district_hospital',
     name: 'DC',
-  },
-  {
+  });
+
+  const contact = personFactory.build({
     _id: 'contact',
     type: 'person',
```

Comment:
```suggestion
```

You can remove this line because the `type: 'person'` is already assigned in the factory - [see here](https://github.com/medic/cht-core/blob/master/tests/factories/cht/contacts/person.js#L27)